### ISSUE TITLE
What about recursive restoration by time of deletion?

### ISSUE THREAD

**Issue by vojta0001** [rubysherpas/paranoia#260]
Hi. 
I was thinking and in my point of view, the recursive restoring by time would be more useful.
In example, when user deletes some records and then admin deletes the user for any reason, It should be better to recursive restore user into state before deletion instead of restore user with all his records even though some records were deleted by user earlier.

**Comment by Jerph** [comment]
Agree. This is a current feature of the acts_as_paranoid gem, with a default of a 2 minute time window for dependent restore.

**Comment by radar** [comment]
Then use acts_as_paranoid if you want this feature?
